SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/GenerateSchemaAndSnapshotsCommandlet.cpp

```diff
@@ -9,6 +9,7 @@
 #include "Engine/World.h"
 #include "FileHelpers.h"
 #include "Misc/Paths.h"
+#include "Engine/LevelStreaming.h"
```

Comment:
Please sort these headers alphabetically.

Reply:
Done in 2d99d9521a2c41431efee332a57962be227f862e